### ISSUE TITLE
UI, German

### ISSUE THREAD

**Issue by Oleg-Chashko** [AdguardTeam/AdGuardVPNForiOS#173]
v.2.0.4(9)

- At the first installation:

![IMG_0612](https://user-images.githubusercontent.com/62497891/181913895-abfc9d50-fb7a-42ac-9f64-31c615d45aec.PNG)

---

- In the process of using the application:

![IMG_0613](https://user-images.githubusercontent.com/62497891/181913806-9d323925-2f84-41f1-a834-719a49df65d7.PNG)

**Comment by mozartinio** [comment]
@Oleg-Chashko
Hello!
As for the first screenshot - is it happening everytime after first installation? (asking because we checked - everything is fine)
As for the second - probably you already have AdGuard (AdBlocker) installed (or no?) that is why integrated mode was already chosen by default in order to work with AdGuard VPN.
(if you dont have AdGuard please sent your logs to us please)
 
Here's what we need you to do:

Open Settings --> General --> Turn Advanced mode on --> Advanced settings --> Turn Debug logs switch on;
Reproduce the issue and remember the exact time when it happened;
Disable "DNS Privacy";
Open the settings --> Support --> Export logs;
System popup will be opened --> Choose "Mail" app
Enter apple@adguard.com as a recipient;
Mention this issue number in the subject and add the time the issue was reproduced at.

**Comment by Oleg-Chashko** [comment]
@mozartinio Hello!

_As for the first screenshot - is it happening everytime after first installation? (asking because we checked - everything is fine)_

- Yes. Perhaps it is because it is an iPhone 5s (because of the small size of the diagonal). The iOS itself has the same artifacts due to the use of German localization and the small size of the diagonal.

- The second screenshot I showed that later on everything is fine.

P.S.: This probably can't be fixed. As it is a bug of the Apple SDK. But I had to let you know that.
